extmod/modusocket.c

```diff
@@ -42,9 +42,20 @@
 
 STATIC const mp_obj_type_t socket_type;
 
+STATIC void socket_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind) {
+    mod_network_socket_obj_t *self = MP_OBJ_TO_PTR(self_in);
+    #if MICROPY_PY_USOCKET_EXTENDED_STATE
+    mp_printf(print, "<socket fd=%d timeout=%d domain=%d type=%d proto=%d bound=%b>",
+        self->fileno, self->timeout, self->domain, self->type, self->proto, self->bound);
+    #else
+    mp_printf(print, "<socket fd=%d domain=%d type=%d proto=%d bound=%b>",
+        self->fileno, self->domain, self->type, self->proto, self->bound);
+    #endif
+}
+
 // constructor socket(family=AF_INET, type=SOCK_STREAM, proto=0, fileno=None)
```

Comment:
Please remove the `fileno` arg from the comment.

Reply:
Fixex